orangecontrib/timeseries/widgets/owdifference.py

```diff
@@ -43,13 +46,20 @@ class Outputs:
     def __init__(self):
         self.data = None
         box = gui.vBox(self.controlArea, 'Differencing')
+
+        gui.radioButtons(box, self, 'calc_difference', 
+                        ['Quotient', 'Difference'], label='Calculate :',
```

Comment:
I prefer 'Compute:', but whatever. :)
Why not also add 'Percant Change', equal to `quotient_values - 1`?

Reply:
+1, on both points.